sklearn/metrics/tests/test_classification.py

```diff
@@ -323,6 +322,7 @@ def test_cohen_kappa():
     assert_almost_equal(cohen_kappa_score(y1, y2), .8013, decimal=4)
 
 
+@ignore_warnings
```

Comment:
`RuntimeWarning: Degrees of freedom <= 0 for slice`
np.corrcoef raises a warning when dividing by zero, and return a NaN.
This test verifies that `matthews_corrcoef` transform the NaN in zeros, so we can silence the warning